src/apps/Map/components/MapSelectorBar/index.js

```diff
@@ -3,28 +3,57 @@ import React, { PureComponent } from 'react';
 import styled from 'styled-components';
 
 import Brace from '~/apps/Map/components/Brace';
-import { orientation } from '~/styles/utils';
+import { orientation, media } from '~/styles/utils';
 
 import HBILegend from './HBILegend';
 import PlanningLegend from './PlanningLegend';
 
+let embedModeActive;
```

Comment:
This looks very unusual in a React Component. I think this should be a state of a Component which is then "transfered" to `styled` via props. Lets have a Chat to see if that would work.

Reply:
Yeah you're right. I fixed it :)